### DISCUSSION TITLE
Splash Screen Customisation

### DISCUSSION THREAD

**Discussion by euroblaze** [open-webui/open-webui#2946] [General]
Hello, 
thank you for the great too and a great community building around it!

This is probably a rookie question from a business user trying to deploy the OpenWebUI for my specific audience.

1. How can the splash-screen questions be customised?
2. Behind each of them, is it possible to augment/preload a contextually rich prompt?

## Example Use Case

I am a regional tour operator helping tourists spend their time and money most effectively when they are visiting my city.
I want my visitor's chat experience to start with the following questions.

1. List the top 10 sight-seeing locations, with website URL and phone numbers.
2. Which restaurants offer the best local food experience?
3. What are the best bars?
4. Where can I go dancing on weekends?
5. List co-working locations with public transportation.
6. ..

Also cool would be to preface the questions with an icon/thumbnail.

![image](https://github.com/open-webui/open-webui/assets/7826363/544c14dc-f76e-448f-9c29-7cc1a3a8c335)

Thank you
Ashant

**Comment by delfireinoso** [comment]
You can edit them in the **settings** page, under **Interface**, **Default Prompt Suggestions**

You can add translate, delete them, and pull a new set
